File: C/lanchonete.c
#include <stdio.h>

int main()
{
    int cod, qtd;
    double valor;

    printf("Codigo do produto comprado: ");
    scanf("%d", &cod);
    printf("Quantidade comprada: ");
    scanf("%d", &qtd);

    switch (cod){
    case 1:
        valor = qtd * 5.0;
        break;

    case 2:
        valor = qtd * 3.5;
        break;

    case 3:
        valor = qtd * 4.8;
        break;

    case 4:
        valor = qtd * 8.9;
        break;

    case 5:
        valor = qtd * 7.32;
        break;
    }

    printf("Valor a pagar: R$ %.2lf", valor);
    return 0;
}

File: Python/lanchonete.py

cod = int(input("Codigo do produto comprado: "))
if cod > 0 and cod < 6:
    qtd = int(input("Quantidade comprada: "))

if cod == 1:
    valor = 5.0 * qtd
elif cod == 2:
    valor = 3.5 * qtd
elif cod == 3:
    valor = 4.8 * qtd
elif cod == 4:
    valor = 8.9 * qtd
elif cod == 5:
    valor = 7.32 * qtd
else:
    print("CODIGO DO PRODUTO INVÁLIDO, ESCOLHA UM CODIGO DE 1 a 5")
    exit()

print(f"Valor a pagar: {valor:.2f}")